src/layout.ts

```diff
@@ -101,6 +108,42 @@ export function layoutParagraph(paragraph: Paragraph, box: Box, fonts: Font[]):
   };
 }
 
+function layoutColumns(block: Columns, box: Box, fonts: Font[]) {
+  const fixedWidth = block.width;
+  const fixedHeight = block.height;
+  const maxWidth = fixedWidth ?? box.width;
+  const maxHeight = fixedHeight ?? box.height;
+  const colWidths = block.columns.map((column) =>
+    column.width == null
+      ? undefined
+      : column.width + (column.margin?.left ?? 0) + (column.margin?.right ?? 0)
+  );
+  const reservedWidth = colWidths.reduce((p, c) => p + (c ?? 0), 0);
+  const nFlexCols = colWidths.reduce((p, c) => p + (c == null ? 1 : 0), 0);
```

Comment:
I didn't get what this is exactly when I first saw the name. How about `numFlexCols` or `flexColsCount`?

Reply:
You're right. Renamed to `flexColCount`.